packages/lottie/src/Lottie.tsx

```diff
@@ -0,0 +1,108 @@
+import type {AnimationItem} from 'lottie-web';
+import lottie from 'lottie-web';
+import type {CSSProperties} from 'react';
+import {useEffect, useRef, useState} from 'react';
+import {continueRender, delayRender, useCurrentFrame} from 'remotion';
+import type {LottieAnimationData} from './types';
+import {getNextFrame} from './utils';
+import {validateLoop} from './validate-loop';
+import {validatePlaybackRate} from './validate-playbackrate';
+
+export interface LottieProps {
```

Comment:
Are types preferred in the Remotion codebase?

Reply:
It doesn't really matter IMO! So this is fine for me